assets/js/app.js

```diff
@@ -80,6 +80,29 @@ $(function() {
   })
 });
 
+// download page os buttons
+
+$(function() {
+  const contentAvailable = "Fork the O3DE source and build the engine and tools directly from any branch. For more details, read the documentation.";
+  const contentUnavailable = "Coming soon.";
+  $(".download-os-button").click(function(event){
+    event.preventDefault()
+    
+    $(".download-os").removeClass("active");
+    $(this).parent().addClass("active");
+
+    if($(this).text().includes("Windows")){
+      $(".download-os-content").text(contentAvailable);
```

Comment:
Windows will also have a binary installer available as part of this update - there should be a button for both "Fork from GitHub" and one for "Download Installer"